irods_specific_implementation/irods_rest_query_api_implementation.h

```diff
@@ -2,9 +2,7 @@
 #include "irods_rest_api_base.h"
 
 #include "irods_query.hpp"
-#include "irods_logger.hpp"
-
-using logger = irods::experimental::log;
+#include "irods_log.hpp"
```

Comment:
This doesn't necessarily need any of the logging facilities provided by iRODS. Given that it is a client, it could use spdlog to manage its own log file or speak to syslog.

Reply:
Is there anything in the implementation file that makes use of things in the **irods_log.hpp** header?

Reply:
i don't see it.  perhaps it was for development... and is a leftover.   just remote it?  @jasoncoposky?

Reply:
torching.